src/main/java/hudson/plugins/performance/PerformancePublisher.java

```diff
@@ -335,7 +323,6 @@ public PerformancePublisher(String sourceDataFiles,
 
         this.nthBuildNumber = nthBuildNumber;
         this.configType = configType;
-        PerformancePublisher.optionType = configType;
```

Comment:
Though honestly, I should have just removed it entirely since it serves no purpose...